packages/dcos-integration-test/extra/conftest.py

```diff
@@ -177,6 +177,18 @@ def _wait_for_srouter_slaves_endpoints(self):
             assert "id" in data
             assert data["id"] == slave_id
 
+    @retrying.retry(wait_fixed=2000,
+                    retry_on_result=lambda r: r is False,
```

Comment:
[Readability comment] - It's better to have a description variable name here in `lambda r` - Does `r` denote `result`?